.changeset/four-candles-add.md

```diff
@@ -0,0 +1,18 @@
+---
+'@backstage/plugin-tech-insights-backend': minor
+'@backstage/plugin-tech-insights-common': minor
+'@backstage/plugin-tech-insights-node': minor
+'@backstage/plugin-tech-insights': minor
```

Comment:
You can actually make all of these _except_ `-node` into `patch`. Additive changea to a utility API [are exempt](https://backstage.io/docs/overview/versioning-policy#changes-that-are-not-considered-breaking) (doesn't affect consumers).